SimplenoteUITests/History.swift

```diff
@@ -0,0 +1,47 @@
+import XCTest
+
+class History {
+
+    class func close() {
+        guard app.staticTexts[UID.Button.restoreNote].waitForExistence(timeout: minLoadTimeout) else { return }
+        app.buttons[UID.Button.dismissHistory].tap()
+    }
+
+    class func setSliderPosition(position: CGFloat) {
+        app.sliders.element.adjust(toNormalizedSliderPosition: position)
+    }
+
+    class func restoreNote() {
+        app.buttons[UID.Button.restoreNote].tap()
+    }
+}
+
+class HistoryAssert {
+    class func historyShown() {
+        let dismissHistoryButton = app.buttons[UID.Button.dismissHistory]
+        let restoreNoteButton = app.buttons[UID.Button.restoreNote]
+
+        XCTAssertTrue(dismissHistoryButton.waitForExistence(timeout: minLoadTimeout), UID.Button.dismissHistory + buttonNotFound)
+        XCTAssertTrue(restoreNoteButton.waitForExistence(timeout: minLoadTimeout), UID.Button.restoreNote + buttonNotFound)
+    }
+
+    class func historyDismissed() {
+        let dismissHistoryButton = app.buttons[UID.Button.dismissHistory]
+        let restoreNoteButton = app.buttons[UID.Button.restoreNote]
+
+        XCTAssertFalse(dismissHistoryButton.exists, UID.Button.dismissHistory + buttonNotAbsent)
+        XCTAssertFalse(restoreNoteButton.exists, UID.Button.restoreNote + buttonNotAbsent)
+    }
```

Comment:
(This is something really extra, feel free to ignore it.)

Since both methods are quite similar, we can apply the [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle here, and do something like:

```swift
    class func historyShown() {
        HistoryAssert.historyExistence(shouldExist: true, errorMessage: buttonNotFound)
    }

    class func historyDismissed() {
        HistoryAssert.historyExistence(shouldExist: false, errorMessage: buttonNotAbsent)
    }

    class func historyExistence(shouldExist: Bool, errorMessage: String) {
        let dismissHistoryButton = app.buttons[UID.Button.dismissHistory]
        let restoreNoteButton = app.buttons[UID.Button.restoreNote]

        XCTAssertEqual(dismissHistoryButton.waitForExistence(timeout: minLoadTimeout),
                       shouldExist,
                       UID.Button.dismissHistory + errorMessage)
        XCTAssertEqual(restoreNoteButton.waitForExistence(timeout: minLoadTimeout),
                       shouldExist,
                       UID.Button.restoreNote + errorMessage)
    }
```

Reply:
This is great, I'm changing it.